package.json

```diff
@@ -2990,9 +2990,6 @@
         "vscode.docker",
         "vscode.yaml"
     ],
-    "extensionPack": [
-        "docker.docker"
-    ],
```

Comment:
@rcjsuen I wanted to call your attention to this. Removing Docker DX from the extension pack will mean that, once this is released, Docker DX will no longer be automatically installed alongside Microsoft's Docker extension. _However_, if we were to leave it in the extension pack, then when the Docker extension gets uninstalled (as part of this toast notification), it takes Docker DX with it. We anticipate a lot of users opting in to Container Tools, so to leave it would mean a lot of Docker DX uninstalls.